packages/ui/src/utils/pipe-custom-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { getValue } from './get-value';
+import { setValue } from './set-value';
+import { PipeMetadata } from '../models/camel/entities/pipe-overrides';
+
+export const getCustomSchemaFromPipe = (metadata: PipeMetadata) => {
+  const name = getValue(metadata, 'name', '');
+  const annotations = getValue(metadata, 'annotations', {} as Record<string, unknown>);
+  const labels = getValue(metadata, 'labels', {} as Record<string, unknown>);
```

Comment:
Let's set the typing at the definition level, so we don't get an `any`.
```suggestion
  const name: string = getValue(metadata, 'name', '');
  const annotations:  Record<string, unknown> = getValue(metadata, 'annotations', {});
  const labels: Record<string, unknown>= getValue(metadata, 'labels', {});
```